services/masterdata/src/lib/core/adapters/masterdata-model.adapter.ts

```diff
@@ -2,17 +2,19 @@
 import { IAdapter } from "../interfaces";
 import { Masterdata } from "../models";
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
 export class MasterdataModelAdapter implements IAdapter<any, Masterdata> {
-  adapt(source: any): Masterdata {
-    return {
-      id: source?.id,
-      masterdataTypeId: source?.masterdataTypeId,
-      masterdataTypeName: source?.masterdataTypeName,
-      key: source?.key,
-      value: source?.value,
-      name: source?.name,
-      description: source?.description,
-      metadata: source?.metadata,
-    };
-  }
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	adapt(source: any): Masterdata {
```

Comment:
@dageorgero  Is source `any`?

Reply:
i didn't use NSwag to generate the angular api classes. i don't have a model for the data retrieved from the backend api.